starter-code/scripts/views/articleView.js

```diff
@@ -83,7 +88,9 @@
      });
    }; */
 
-  // COMMENT: What does this method do?  What is it's execution path?
+  /* COMMENT: What does this method do?  What is it's execution path?
+  we create a method of articleView called index that takes articles as a parameter. It then uses a jquery selector to get any element with the ID of articles and shows it but hides its siblings. it then remove any article element with the ID articles. It then loops through the articles array appending the resault of the render function given the argument of a to any elements with the articles ID. It then calls the poopulateFilters method and the handleFilters methon of the articleView object. Lastly if any article element with the ID of articles has a length greater than 1 hide everything after the first two elements in the class article-body.
+  */
```

Comment:
Are you sure '#articles article' selects article elements with the ID of articles?

If I were to have the selector "div p", what would I be targeting?